README.md

```diff
@@ -8,13 +8,22 @@ Command-line tool for validating a given XML file against a specific XSD Schema.
 
 ### Installing
 
+Note: The tool requires Python 3.6+
+
 Clone the project and install with `pip`:
 ```
 git clone https://github.com/CSCfi/metadata-submitter-tools.git
 cd metadata-submitter-tools
 pip install .
 ```
 
+Note: If running the tool results in a `RuntimeError` about Python 3 using ASCII as encoding,
+you will need to set the following environment variables:
+```
+export LC_ALL=en_US.utf-8
```

Comment:
I think we can set the environment variables for the user ourselves 
like os.environ["LC_ALL"] = "en_US.utf-8" and to unset it
https://docs.python.org/3.7/library/os.html#os.unsetenv or `del`